src/components/textarea/bl-textarea.css

```diff
@@ -0,0 +1,131 @@
+:host {
+  display: inline-block;
+  width: 200px;
+  position: relative;
+
+  --bl-textarea-padding-vertical: var(--bl-size-2xs);
+  --bl-textarea-padding-horizontal: var(--bl-size-xs);
+  --bl-textarea-border-color: var(--bl-color-border);
+  --bl-textarea-icon-color: var(--bl-color-content-tertiary);
+  --bl-textarea-text-color: var(--bl-color-content-primary);
```

Comment:
Those variables should be kept internal. So

- [ ] Move them away from `:host` element since host element can be set outside of shadowdom. You can create a wrapping element if you need.
- [ ] remove `bl-textarea` prefixes.